ct-aa/dotTheory---TypeScript
# JS-961: Added The Course Project Setup

# Related Tickets
[JS-961](https://codeandtheory.atlassian.net/browse/JS-961)

# Description
- Added the Course Project Setup.

# Validation
- Open the terminal.
- Run `npm run start`.
- Inspect the web page that will be open.
- The console log test should appear.
<img width="1755" alt="Screen Shot 2023-10-04 at 11 30 00 AM" src="https://github.com/ct-aa/dotTheory---TypeScript/assets/93116252/27a1fc25-164c-4e8c-a3a0-e427c97abec4">

# Risks
N/A
